src/main/java/com/minecolonies/util/LanguageHandler.java

```diff
@@ -62,8 +63,10 @@ public static String getString(String key)
      * @param defaultValue the value to return if no key is found
      * @return Localized string
      */
+    @SuppressWarnings("deprecation")
     public static String getString(String key, String defaultValue)
     {
+        //todo: use TextComponentTranslation like mojang wants us to
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)